<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICCutModuleActionPage</class>
 <widget class="QWidget" name="ICCutModuleActionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="4">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Fixture:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>X1:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="zSpeed">
     <property name="text">
      <string>80</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="ICComboBox" name="fixtureSel">
     <item>
      <property name="text">
       <string>Fixture-1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fixture-2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fixture-3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fixture-4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sucker-1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sucker-2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sucker-3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sucker-4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Reserve-1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Reserve-2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Reserve-3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Reserve-4</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="4">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="fixtureDelay">
     <property name="text">
      <string>0.50</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="y1UpPos">
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
   </item>
   <item row="9" column="4">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Y1:Up</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="zPos">
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="x1Pos">
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="y1Pos">
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="y1Speed">
     <property name="text">
      <string>80</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="x1Speed">
     <property name="text">
      <string>80</string>
     </property>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="y1UpSpeed">
     <property name="text">
      <string>80</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="setIn1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Set In</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Cut Outlet</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="5">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Y1:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QPushButton" name="setIn2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Set In</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
  <customwidget>
   <class>ICComboBox</class>
   <extends>QComboBox</extends>
   <header>iccombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
